favorites.py

```diff
@@ -179,10 +198,11 @@ def load_favorite_files(cls, obj, force=False, clean=False, win_id=None):
         # Is project enabled
         FavProjects.project_adjust(obj, win_id, force)
 
+        cls.check_aliases(obj.file_name)

```

Comment:
I added the check here so that it's run indifferently for global and project json files